Error creating commit: Invalid request

```
[INFO] --- site-maven-plugin:0.9:site (execution1) @ lolchatapi ---
[INFO] Creating 18 blobs
[INFO] Creating tree with 19 blob entries
[INFO] Merging with tree 9a11f1f5c5d4e3441cb3949dcf9e907fa6c1f1d0
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 20.494 s
[INFO] Finished at: 2014-08-29T12:09:59+02:00
[INFO] Final Memory: 31M/267M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.github.github:site-maven-plugin:0.9:site (execution1) on project lolchatapi: Error creating commit: Invalid request.
[ERROR] 
[ERROR] Expected data to be of type "object"; value was: nil.
[ERROR] Expected data to be of type "object"; value was: nil. (422)

Expected data to be of type "object"; value was: nil.
Expected data to be of type "object"; value was: nil. (422)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:216)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:153)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:145)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:116)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:80)
	at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:51)
	at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:120)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:347)
	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:154)
	at org.apache.maven.cli.MavenCli.execute(MavenCli.java:584)
	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:213)
	at org.apache.maven.cli.MavenCli.main(MavenCli.java:157)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:289)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)
	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)
	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)
	at org.codehaus.classworlds.Launcher.main(Launcher.java:46)
Caused by: org.apache.maven.plugin.MojoExecutionException: Error creating commit: Invalid request.

Expected data to be of type "object"; value was: nil.
Expected data to be of type "object"; value was: nil. (422)
	at com.github.maven.plugins.site.SiteMojo.execute(SiteMojo.java:440)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:132)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:208)
	... 20 more
Caused by: org.eclipse.egit.github.core.client.RequestException: Invalid request.

Expected data to be of type "object"; value was: nil.
Expected data to be of type "object"; value was: nil. (422)
	at org.eclipse.egit.github.core.client.GitHubClient.createException(GitHubClient.java:552)
	at org.eclipse.egit.github.core.client.GitHubClient.sendJson(GitHubClient.java:643)
	at org.eclipse.egit.github.core.client.GitHubClient.post(GitHubClient.java:757)
	at org.eclipse.egit.github.core.service.DataService.createCommit(DataService.java:397)
	at com.github.maven.plugins.site.SiteMojo.execute(SiteMojo.java:434)
	... 22 more
```

Everything worked great 5 days ago, but today I started getting this error. Is there anything on my end that I can do?

--- a/github-site-plugin/src/main/java/com/github/maven/plugins/site/SiteMojo.java
+++ b/github-site-plugin/src/main/java/com/github/maven/plugins/site/SiteMojo.java
@@ -433,7 +433,7 @@ public class SiteMojo extends GitHubProjectMojo {
 
             CommitUser author = new CommitUser();
             author.setName(user.getName());
-            author.setEmail(user.getEmail());
+            author.setEmail(userService.getEmails().get(0));
             author.setDate(new GregorianCalendar().getTime());
 
             commit.setAuthor(author);

